workflow.rst

```diff
@@ -799,6 +799,10 @@ of domain logic in your templates:
 
 ``workflow_transitions()``
```

Comment:
I would, for all functions in this list, inline the parameters in the function name:

```suggestion
``workflow_transitions(object $subject, string $name = null)``
```

I think this is the most clear and concise way of documenting them.